spec/support/encoding_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module EncodingHelper
```

Comment:
Might be good to document the module a bit, so it's clearer what we're aiming to achieve with it.

Reply:
👍